source/manual/browser-support.html.md

```diff
@@ -5,7 +5,7 @@ parent: "/manual.html"
 layout: manual_layout
 section: Frontend
 type: learn
-last_reviewed_on: 2019-10-08
+last_reviewed_on: 2019-10-12
```

Comment:
Only if I would've updated the year too 🤦

Reply:
Aw wow I definitely should have spotted that too 🤦